WooCommerce/WooCommerceTests/ViewRelated/MainTabBarControllerTests.swift

```diff
@@ -59,21 +58,20 @@ final class MainTabBarControllerTests: XCTestCase {
 
         // Assert
         XCTAssertEqual(tabBarController.viewControllers?.count, 4)
-        assertThat(tabBarController.tabNavigationController(tab: .myStore, isHubMenuFeatureFlagOn: isHubMenuFeatureFlagOn)?.topViewController,
+        assertThat(tabBarController.tabNavigationController(tab: .myStore)?.topViewController,
                    isAnInstanceOf: DashboardViewController.self)
-        assertThat(tabBarController.tabNavigationController(tab: .orders, isHubMenuFeatureFlagOn: isHubMenuFeatureFlagOn)?.topViewController,
+        assertThat(tabBarController.tabNavigationController(tab: .orders)?.topViewController,
                    isAnInstanceOf: OrdersRootViewController.self)
-        assertThat(tabBarController.tabNavigationController(tab: .products, isHubMenuFeatureFlagOn: isHubMenuFeatureFlagOn)?.topViewController,
+        assertThat(tabBarController.tabNavigationController(tab: .products)?.topViewController,
                    isAnInstanceOf: ProductsViewController.self)
-        assertThat(tabBarController.tabNavigationController(tab: .reviews, isHubMenuFeatureFlagOn: isHubMenuFeatureFlagOn)?.topViewController,
-                   isAnInstanceOf: ReviewsViewController.self)
+        assertThat(tabBarController.tabNavigationController(tab: .hubMenu)?.topViewController,
+                   isAnInstanceOf: HubMenuViewController.self)
     }
 
     func test_tab_view_controllers_returns_expected_values_with_hub_menu_enabled() {
```

Comment:
super nit: maybe this can be renamed?
```suggestion
    func test_tab_view_controllers_returns_expected_values() {
```

Reply:
Definitely, renamed: 85464debdd41207208b25ac9c0d7c324c463cdc0